cluster/src/main/java/org/apache/iotdb/cluster/query/LocalQueryExecutor.java

```diff
@@ -1043,7 +1044,12 @@ public ByteBuffer last(LastQueryRequest request)
 
     List<Pair<Boolean, TimeValuePair>> timeValuePairs =
         LastQueryExecutor.calculateLastPairForSeriesLocally(
-            partialPaths, dataTypes, queryContext, expression, request.getDeviceMeasurements());
+            partialPaths,
+            dataTypes,
+            queryContext,
+            expression,
+            request.getDeviceMeasurements(),
+            IoTDBDescriptor.getInstance().getConfig().isEnableIDTable());
```

Comment:
Is there any test based on this parameter? Maybe the function implement could access the config via IoTDB global  variables.

Reply:
Sure~ Fixed~